### PR TITLE
Adds valid PAT testing to existing ec2 pipes

### PR DESCRIPTION
**Pull request by supertopher**
adds the same workflow seen in gradle.yml to other pipelines

## What
grade.yml is running through PAT keys to check for valid keys.  lets do that elsewhere

## How
adds a need before GitHub token invoking the supertopher branch

### CODE REVIEW

File: .github/workflows/gke-kube-test-command.yml [hunk 1]
```diff
@@ -15,10 +15,29 @@ on:
...
+    outputs:
+      pat: ${{ steps.variables.outputs.pat }}
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+      - name: Check PAT rate limits
+        id: variables
+        run: |
+          ./tools/bin/find_non_rate_limited_PAT \
+             ${{ secrets.SUPERTOPHER_PAT }} \
```
Comment (davinchia) [inline .github/workflows/gke-kube-test-command.yml:31]: we should change these before merging this in.

Comment (davinchia) [reply .github/workflows/gke-kube-test-command.yml:31]: same for the rest of the jobs.

**Review by davinchia — APPROVED** [review]
Nice work! Fingers crossed!

File: .github/workflows/github-community-label-bot.yaml [hunk 2]
```diff
@@ -12,4 +12,4 @@ jobs:
       - uses: harshithmullapudi/label-actions@75686c2b3de17244526f10a22424f319d0bc134f
         with:
           github-token: ${{ secrets.OCTAVIA_PAT }}
-          github-username: ${{ secrets.LABEL_BOT_USERNAME }}
+          github-username: octavia-squidington-iii
```
Comment (supertopher) [inline .github/workflows/github-community-label-bot.yaml:15]: Whatever we were using before it's Octavia now, I'm killing these secrets after this

**Review by supertopher — COMMENTED** [review]
LGTM assuming testing passes I'll merge away

**Pull request merged**